marketplace/netlify-build/src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link type="text/css" rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="text/javascript" src="https://cdn.pubnub.com/sdk/javascript/pubnub.4.21.7.min.js"></script>
```

Comment:
can we import it as usual npm dependency?

Reply:
Done